### DISCUSSION TITLE
Plane doesn't follow plan and ILS problem

### DISCUSSION THREAD

**Discussion by rpoulachon** [flybywiresim/aircraft#4696] [General]
Hi everybody. I use the latest devlopment branch and i have had some curious behaviours.i have established a plan from LFBO to LFMN with simbrief without any problems.
During the flight, without any reasons, the flight plan has been erased (no line between the waypoint) and my plane has completely lost his road.
Have you noticed this ?

Thank you and sorry for my poor english.
Best regards

**Comment by ExampleWasTaken** [comment]
This is most probably caused by the default Asobo flight plan manager the A32NX still uses. A new custom one is currently in development.

**Reply by rpoulachon** [reply]
Thank you for your response.
But I did no have such problems in previous version (development branch). I could do flight without any unforeseen events.
That why I suppose there are changes made in the last release (new flight plan model ?).

Thank you again.

Best regards.

**Reply by ExampleWasTaken** [reply]
The behavior of the current flight plan manager is pretty much non predictable. Therefore you might have been lucky before, and it always behaved as it should.

**Comment by Frecu5** [comment]
I have also had the same ILS issue with the latest stable version 0.6.1.
On two IFR flights:
LGKR-LICJ
LICJ-LIRN
The plane does not detect the ILS glide path when landing.
In the previous version, this did not happen.

**Reply by ExampleWasTaken** [reply]
Please reproduce the issue and post a screenshot of your primary flight display, while the issue occurs. 
I would also need to know the airport and runway your landing at.

**Reply by Frecu5** [reply]
The airports have already been reflected in my previous mail: LICJ and LIRN.

**Reply by ExampleWasTaken** [reply]
> 
> 
> The airports have already been reflected in my previous mail: LICJ and LIRN.

Without a screenshot I can't do anything for you.

**Reply by Frecu5** [reply]
![Microsoft Flight Simulator 1](https://user-images.githubusercontent.com/83861493/117565727-156c8480-b0b3-11eb-83cb-923262fbe208.jpg)
![Microsoft Flight Simulator 2021-05-08 19-59-23_Moment(2)](https://user-images.githubusercontent.com/83861493/117565731-1d2c2900-b0b3-11eb-89ce-fd36ffa9dbf0.jpg)
![Microsoft Flight Simulator 2021-05-08 19-59-23_Moment(3)](https://user-images.githubusercontent.com/83861493/117565732-20bfb000-b0b3-11eb-8252-3d8b8ff782a2.jpg)
![Microsoft Flight Simulator 2021-05-08 19-59-23_Moment(4)](https://user-images.githubusercontent.com/83861493/117565738-24ebcd80-b0b3-11eb-9758-9a0f13ca66e0.jpg)
![Microsoft Flight Simulator 2021-05-08 19-59-23_Moment(5)](https://user-images.githubusercontent.com/83861493/117565742-27e6be00-b0b3-11eb-8156-a360640b9f7e.jpg)

**Reply by Frecu5** [reply]
Hello.
Attached are some catches of the approach to Capodichino Airport (LIRN).
As you can see, the automatic pilot does not attend to the ILS.

Best regards

**Reply by solarisfire** [reply]
Looks like it catches the Localizer but you're above the glideslope at that point so it's never captured...

In this screenshot:

![image](https://user-images.githubusercontent.com/420952/117568060-9a0cd200-b0b6-11eb-9d81-1894a99d4840.png)

You're at 4500ft.

But according to all the charts:

![image](https://user-images.githubusercontent.com/420952/117568100-b872cd80-b0b6-11eb-91bd-e65aa8bbf49b.png)

Glideslope capture should be around 3770ft 10 miles out...

ILS will never dive to capture a g/s, you need to be under it with APPR mode engaged and it will capture.

**Reply by solarisfire** [reply]
You're passing POM which should be at 1800ft.

**Reply by solarisfire** [reply]
It's tricky without seeing the approach plugged into the FMGS, but are you flying the ILS W? That's a weird approach that is rarely used, and probably never used by an A320 but pretty fun in sim.

![image](https://user-images.githubusercontent.com/420952/117568357-ba895c00-b0b7-11eb-92ff-e1610d04586a.png)

But still, you should be at 3700ft 7 miles from POM and cross POM at 1800ft.

**Reply by Frecu5** [reply]
Thank you for your altitude indications.
I'll repeat the flight with your directions, I'm sure everything's fine, however, the FS2020 ATC never tells me to drop altitude after 4,500 feet and I think I should.

I'll tell you.....

**Reply by Frecu5** [reply]
Hello...
I've repeated the flight with your altitude indications and it just came out correct.
My mistake was, waiting for the ATC to order me to drop altitude.
In the future, I will listen to the PFD and the Letters.
I really appreciate your help.
Greetings.

**Reply by solarisfire** [reply]
No worries at all :-)

I can very much recommend using the appropriate charts when flying procedures like this. It gives you all of the altitude information that you need.

Happy flying :-)

**Comment by AdamNZ** [comment]
ATC in MSFS is a total joke. I would ignore it completely and make your descents yourself - using the CONSTRAINTS option in the PFD (and any additional info you might be able to find in approach charts etc.).

I rarely have ATC enabled - though I do occasionally try it, just to see if it's got better (no). I don't think I've had a failed intercept in FBW yet. I always intercept from below.

**Reply by Frecu5** [reply]
Thank you for your appreciation.
